tools/cmd/linter/main.go

```diff
@@ -341,6 +341,30 @@ func parseField(fs map[string]Field, field *parser.Field) {
 	fs[snakeToCamel(field.FieldName)] = f
 }
 
+func parseMapField(fs map[string]Field, field *parser.MapField) {
```

Comment:
**❗️Note:** I added the `parseMapField` function to handle protobuf fields of type `map`, otherwise the field cannot be recognized and cannot pass lint-remain